### DISCUSSION TITLE
About the use of rem unit

### DISCUSSION THREAD

**Discussion by lexoyo** [GrapesJS/grapesjs#5713] [Q&A] [answered]
Hello
I'm using rems a lot as it is a good practice to make responsive design process simple, you just change the root element font size and everything follows
But in grapesjs, when the user wants to change the font size of the root element it is impossible to select `<html>` in grapesjs ui am I right? How do you handle this?
Have a nice day

**Comment by artf** [comment, marked as answer]
As Selectors and StyleManager are independent of components, you can create/select the CSS rule you want to update. 

Here is an example 
https://jsfiddle.net/Ln16qgek/
but the UX is up to you, eg. It could be a button trait added to the Wrapper component to allow such a thing.

**Reply by lexoyo** [reply]
Oh that makes sense thank you!
